src/Client.php

```diff
@@ -11,6 +13,11 @@ class Client
      */
     protected $client;
 
+    /**
+     * @var Meema\MeemaApi\Models\Folder
+     */
+    public $folders;
```

Comment:
Let set `$folder` as protected because we dont want that to be available for direct use in the client.